.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 <!--
 Hello and thank you for the issue!
-If you would like to report a bug, we have added some points below that you can fill out.
+If you would like to report a bug, please consider using "Help | Create New Issue" action, so all necessary information will be filled out automatically.
```

Comment:
Let's keep original message and add a note about `Create New Issue` action as a separate sentence. Something like:
```
If you would like to report a bug, we have added some points below that you can fill out.
Consider using "Help | Create New Issue" action that fills out all necessary information automatically.
```